src/components/basic/Table/PageLoadingTable.tsx

```diff
@@ -53,6 +53,7 @@ export interface PageLoadingTableProps<Row, Args>
   allItems?: Row[]
   callbackToPage?: (data: PaginResult<Row>) => void
   allItemsLoadedHint?: string
+  mappedTableSearchTranslation?: Record<string, string>
```

Comment:
@Usmanfee In the existing components we simply pass the message in the current language, so the suggested solution works differently. See my comment in the ticket: https://github.com/eclipse-tractusx/portal-shared-components/issues/368#issuecomment-2482384768